web/viewer.css

```diff
@@ -1540,7 +1542,7 @@ dialog :link {
 }
 
 .visibleMediumView {
-  display: none;
+  display: none !important;
```

Comment:
`!important` here and below is necessary because the rules inside a scope take precedence over rules outside of a scope, and within the secondary toolbar scope we now set the `display` property. Using `!important`  here should overall be safer to ensure that this always takes precedence for correct display in the viewer, but especially now that we're using CSS scoping more often it will also avoid issues for future patches.